riscv64/start_in_qemu.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -ex
+
+$QEMU_DIR/bin/qemu-system-riscv64 \
+    -M virt,aclint=on,aia=aplic-imsic -nographic \
+    -smp 16 -m 32G \
```

Comment:
Perhaps we risk having a resource overcommit here. Should we first check how many cpu and how much ram the host has?

Reply:
> I just checked what type of AWS instances our CI is using (c5d.metal) and how many buildkite agents we pack per metal (5), so here we can use
> 
> and still have a bit of vcpu and memory to spare outside of the VMs :)

Originally I was using `-smp 3 -m 6G` here, while this change is suggested by @roypat.

Reply:
Okay, but for testing new changes, we also build it locally.
So, maybe we should add a parameter or use `nproc` to tune it and leave the default low enough to run on any workstation.

Reply:
Maybe something like max(nproc/5, 3) (and similar for memory), with a comment about where those numbers come from? I'd really like to use as much of the .metal as possible, but argument for testing locally is a good one!